kvk/Issues/official/description.xml

```diff
@@ -129,6 +129,16 @@
 				<nl>De naam van de maatschappelijke activiteit</nl>
 			</Description>
 		</Attribute>
+		<Attribute id="tradeNames" optional="true">
+			<Name>
+				<en>Trade names</en>
+				<nl>Handelsnamen</nl>
+			</Name>
+			<Description>
+				<en>The trade names under which the company trades</en>
+				<nl>De handelsnamen waar de onderneming onder handelt</nl>
+			</Description>
+		</Attribute>
```

Comment:
Newly added attributes should always be at the end of the `Attributes` list.

Reply:
OK, I moved the new attribute to the end of the list

Reply:
Nice, thanks. You forgot to re-sign after your changes. Can you still do that?

Reply:
forgot about this indeed, I have added the signature now